fixtures/credhub-test-app/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/disaster-recovery-acceptance-tests/fixtures/credhub-test-app
+
+go 1.14
```

Comment:
Could we bump this version to the latest? I know it's only a test app but nonetheless...

Reply:
yup, that's coming later with the automatic depedency bumping stories :)